pkts-core/src/test/java/io/pkts/framer/EthernetFramerTest.java

```diff
@@ -40,4 +44,14 @@ public void testEthernetFramerNoParent() throws Exception {
         this.framer.frame(null, this.ethernetFrameBuffer);
     }
 
+    @Test
+    public void testAcceptsAllEtherTypes() throws Exception {
+        for (EthernetFramer.EtherType t : EthernetFramer.EtherType.values()) {
+            Buffer frame = this.ethernetFrameBuffer.slice();
+            frame.setByte(12, t.b1);
```

Comment:
doesn't compile. Private access. I guess I was a little too fast accepting this pull request :-). I'll fix it with a quick update.